app/src/main/java/com/android/sample/ui/dialogs/FilterDialog.kt

```diff
@@ -68,11 +68,11 @@ fun FilterDialog(
                     horizontalArrangement = Arrangement.SpaceBetween,
                     verticalAlignment = Alignment.CenterVertically) {
                       // Minimum Price Display
-                      Text(text = "€ 0,0", modifier = Modifier.testTag("minPriceDisplay"))
```

Comment:
simple change in currency from € to CHF in the filtering options of filterDialog. LGTM!